pages/404.js

```diff
@@ -0,0 +1,381 @@
+import * as React from "react";
+import * as Constants from "~/common/constants";
+
+import { css } from "@emotion/react";
+
+import WebsitePrototypeWrapper from "~/components/core/WebsitePrototypeWrapper";
+
+const STYLES_ROOT = css`
+  width: 80%;
+  margin-left: auto;
+  margin-right: auto;
+`;
+
+const STYLES_404 = css`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  min-height: 60vh;
+  h1 {
+    font-size: 120px;
+  }
+`;
+
+const STYLES_NAV = css`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 88px 0px;
+  font-family: ${Constants.font.text};
+`;
+
+const STYLES_NAVLINKS = css`
+  display: flex;
+  a{
+    margin-left: 20px;
+    text-decoration none;
+  }
+  `;
+
+const STYLES_GLITCH = css`
+  ::before,
+  ::after {
+    right: 0;
+    margin: auto;
+    content: "404";
+    position: absolute;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    min-height: 80vh;
+    min-width: 50vh;
+    overflow: hidden;
+    background-color: #f7f7f7;
+    color: #000;
+  }
+  ::before {
+    left: 8px;
+    text-shadow: 2px 0 #00ffea;
+    animation: glitch 3s infinite linear;
+  }
+  ::after {
+    left: 8px;
+    text-shadow: -2px 0 #fe3a7f;
+    animation: glitch 2s infinite linear;
+  }
+  @-moz-keyframes glitch {
```

Comment:
We don't need `vendor` specific prefixes because we auto prefix thanks to emotion. can you take these out?